docs/install.rst

```diff
@@ -63,14 +63,32 @@ packages install config files and systemd unit files. The dashboard is installed
 package.
 
 
-With pip
----------
-Inmanta can be installed with pip.
+Linux, Mac and Windows with pip install
+---------------------------------------
+Inmanta published new releases on `PyPI <https://pypi.org/project/inmanta/>`_. You can use the built-in python package manager
+pip to install Inmanta. This should work for all Linux operating systems with Python >= 3.6. The install also works on
+Windows: the compile and export functions. Server and agent is not support for now. Inmanta requires many dependencies so it
+is recommended to use a virtual env to install inmanta. Next to python, it requires git to be available as well.
 
 .. code-block:: sh
+    :caption: Install on Linux (and Mac)
+
+    # Install python3 >= 3.6 and git
+    python3 -m venv /opt/inmanta
+    /opt/inmanta/bin/pip install inmanta
+    /opt/inmanta/bin/inmanta --help
+
 
-    pip install inmanta
+For Windows the commands are similar, except only compile and export are support. This is useful in the :ref:`push-to-server`
```

Comment:
If the command is different, for windows I would make a new heading (easier for readers, just jump to right section, follow all steps sequentially)

Reply:
perhaps add links on to of the page to the preferred method on each platform